src/Analysis/Engine/Impl/Infrastructure/PathUtils.cs

```diff
@@ -382,6 +408,46 @@ static class PathUtils {
             return null;
         }
 
+        public static bool IsNormalizedPath(string path) {
+            var separator = Path.DirectorySeparatorChar;
+            int offset;
+            if (path.Length >= 1 && path[0] == separator) {
+                offset = 1;
+            } else if (IsWindows && path.Length >= 2 && IsValidWindowsDriveChar(path[0]) && path[1] == ':') {
+                offset = 2;
+            } else {
+                return false;
+            }
+
+            var impossibleName = false;
+            for (var i = offset; i < path.Length; i++) {
+                var character = path[i];
+                if (character == Path.AltDirectorySeparatorChar) {
```

Comment:
`Path.AltDirectorySeparatorChar == Path.DirectorySeparatorChar` on Unix

Reply:
There is some fun in `PathEqualityComparerTests`

Reply:
> `Path.AltDirectorySeparatorChar == Path.DirectorySeparatorChar` on Unix

Didn't know. Thanks!